lib/bokken/accounts/guardian.ex

```diff
@@ -0,0 +1,22 @@
+defmodule Bokken.Accounts.Guardian do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  @primary_key {:id, :binary_id, autogenerate: true}
+  @foreign_key_type :binary_id
+  schema "guardians" do
+    field :city, :string
+    field :mobile, :string
+
+    belongs_to :user, Bokken.Accounts.User, foreign_key: :user_id
+
+    timestamps()
+  end
+
+  @doc false
+  def changeset(guardian, attrs) do
+    guardian
+    |> cast(attrs, [:mobile, :city, :user_id])
+    |> validate_required([:mobile, :city, :user_id])
```

Comment:
I  think city should not be a required param but optional. I also think we should validate it's a valid city (maybe we will need to add country as a combination, maybe an embed struct with country + city). 

I think we should verify that mobile is a valid number with some regex.

I also think we should be consistent between all changesets on the required fiels and optional fields by having module variables with those fields. Something like in [here](https://github.com/coderdojobraga/bokken/blob/db272747c925d718a8a61b4f417662673d2d53e4/lib/bokken/accounts/user.ex#L9).